fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/FedoraResourceImplTest.java

```diff
@@ -395,6 +398,14 @@ public void testGetProperty() throws RepositoryException {
     }
 
     @Test
+    public void testSetURIProperty() throws URISyntaxException, RepositoryException {
+        final String prop = "premis:hasEventRelatedObject";
+        final String uri = "http://localhost:8080/rest/1";
```

Comment:
The argument name and javadoc indicates that the URI is a relative-path:
https://github.com/fcrepo4/fcrepo4/pull/781/files#diff-688c65561dd248cf2a0096c799be5d18R89

This integration test is not a relative-path, and furthermore, the host and port (in a production scenario) may change or need idTranslation.

To be clear, are you expecting Resource.setURIProperty() to use full URLs or relative-paths? If full URLs, then are we concerned about context translation?